src/state/state.ts

```diff
@@ -2,19 +2,42 @@ import type { BigNumber } from 'ethers';
 import { produce, type Draft } from 'immer';
 
 import type { Config } from '../config/schema';
-import type { LocalSignedData, AirnodeAddress, TemplateId } from '../types';
+import type { chainId, DApiName, DecodedDataFeed, DataFeedId, SignedData } from '../types';
 
 interface GasState {
   gasPrices: { price: BigNumber; timestampMs: number }[];
   sponsorLastUpdateTimestampMs: Record<string, number>;
 }
 
+export interface DataFeedOnChainValue {
+  value: BigNumber;
+  timestamp: number; // in seconds
+}
+
+export interface DApi {
```

Comment:
The `DApi` feels to generic. I think we could be more explicit and call it `DapiState`